eslint.config.js

```diff
@@ -0,0 +1,17 @@
+import pluginVue from "eslint-plugin-vue";
+import eox from "@eox/eslint-config";
+
+export default [
+  ...eox,
+  ...pluginVue.configs["flat/essential"],
```

Comment:
we need the flat essentials specifically in eodash? Not sure what that does

Reply:
we need one of the `flat` options in `eslint.config.js` to apply vue linting rules and I chose `essential` that applies the most important rules for the vue code to be correct. Here's the [list of all the flat options](https://eslint.vuejs.org/user-guide/#bundle-configurations-eslint-config-js)